lib/icedfrisby.js

```diff
@@ -664,26 +694,14 @@ class Frisby {
     return this
   }
 
-
   /**
    * @param {string} header - header key
    * @param {string} pattern - header value content regular express
    * @return {object}
-   * @desc HTTP header expect helper regular expression match
+   * @desc Alias for expectHeader
    */
   expectHeaderToMatch (header, pattern) {
```

Comment:
Since we're calling this an alias, probably it should pass `options` through as well.

Reply:
Originally it did, with an explicit `false` but since it was only a backwards-compatibility fix, I decided this should rely on the defaults. I've also removed this method from the docs - we'd prefer to drive new users to the unified way, right?